src/extras/gizmo/gizmo.js

```diff
@@ -25,9 +26,14 @@ const tmpM2 = new Mat4();
 const tmpR1 = new Ray();
 
 // constants
-const MIN_GIZMO_SCALE = 1e-4;
+const LAYER_NAME = 'Gizmo';
+const MIN_SCALE = 1e-4;
 const PERS_SCALE_RATIO = 0.3;
 const ORTHO_SCALE_RATIO = 0.32;
+const UPDATE_EPSILON = 1e-6;
+
+let gizmoLayer = null;
```

Comment:
Will this work with multi-applications on the same page?

Reply:
Just fixed 😅